### PR TITLE
Update configure-end-user-spam-notifications-in-exchange-online.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a document to more information section (PowerShell reference)

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1291

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/configure-end-user-spam-notifications-in-exchange-online.md [hunk 1]
```diff
@@ -90,4 +90,5 @@ You can also use the Security and Compliance Center (SCC) to configure end-user
 ## For more information
 
 [Configure your spam filter policies](configure-your-spam-filter-policies.md)
+[Configure Spam filter using powershell](https://docs.microsoft.com/en-us/powershell/module/exchange/antispam-antimalware/set-hostedcontentfilterpolicy?view=exchange-ps)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-end-user-spam-notifications-in-exchange-online.md]:
```suggestion
[Set-HostedContentFilterPolicy](https://docs.microsoft.com/powershell/module/exchange/antispam-antimalware/set-hostedcontentfilterpolicy?view=exchange-ps)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. Please also have a look at the rich text. This line does not appear to display correctly.

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #1291.

Thank you.

cc: @kenwith

**Pull request merged**